config/config_test.go

```diff
@@ -268,34 +269,6 @@ func TestParsingUserPasswordErrorEnv(t *testing.T) {
 			envPassword))
 		os.Unsetenv(test.envVar)
 	}
-}
-
-func TestHostPortEndpointEnvError(t *testing.T) {
-	tests := []struct {
-		envVar string
-		value  string
-		err    string
-	}{
-		{
-			envVar: envAgentHost,
-			value:  "user",
-			err:    fmt.Sprintf("cannot set env vars %s and %s together", envAgentHost, envEndpoint),
```

Comment:
this error is no longer tested?

Reply:
Yes, we don't need that error any more.
With this PR, if both envEndpoint and envAgentHost are set, envEndpoint will be only used.